packages/components/src/gridicon/index.tsx

```diff
@@ -17,11 +17,46 @@ interface Props {
 
 type AllProps = Assign< React.SVGProps< SVGSVGElement >, Props >;
 
+function isCrossOrigin( url: string ) {
+	try {
+		return new URL( url ).origin !== window.location.origin;
```

Comment:
Using `new URL( url, window.location )` will prevent exceptions on relative URLs. Which _are_ same-origin.

Reply:
Oooh that's a great point!

Reply:
WDYT about this?

```js
return new URL( url, window.location.href ).origin !== window.location.origin;
```

Reply:
Yes, this is better 🙂 

The `try` block is also important for SSR. In that case we just want to use the `url` value without any fetching or checks.